packages/uswds-core/src/styles/settings/_settings-components.scss

```diff
@@ -110,6 +110,15 @@ $theme-identifier-max-width: "desktop" !default;
 $theme-identifier-primary-link-color: default !default;
 $theme-identifier-secondary-link-color: "base-light" !default;
 
+// In-Page Navigation
+$theme-in-page-nav-background-color: "white" !default;
+$theme-in-page-nav-background-padding: 2 !default;
+$theme-in-page-nav-background-radius: "lg" !default;
+$theme-in-page-nav-bar-color: "secondary-dark" !default;
+$theme-in-page-nav-bar-width: 2px !default;
+$theme-in-page-nav-font-family: "ui" !default;
+$theme-in-page-nav-top-position: 8 !default;
```

Comment:
I like that this setting is customizable. 
I wonder if a smaller default might be better to start. 8 units just feels big on a short screen, so I am wondering if 4 might be better to start with?

Reply:
Thanks Amy, yes I started with 4 but then translating that to the USWDS documentation site caused it to be too close to the fixed header. The 8 gives it a good feel, but I can change it in uswds and then customize it in my uswds-site working branch? What do you think?

Here's a screen shot that shows where it stops at when sticky is engaged and 8 (4rem) is used for top:

![image](https://user-images.githubusercontent.com/19338309/185439012-d72e3440-2a02-4c89-b3a0-456ed10442b9.png)

Reply:
It makes sense to me to use a shorter default value and then customize it for uswds-site, since the header nav on the USWDS site seems to be the exception rather than the rule.